### ISSUE TITLE
Using Minimum Risk Training (MRT)

### ISSUE THREAD

**Issue by patelrajnath** [rsennrich/nematus#40]
Hi Rico,

We are trying to use the MRT feature of Nematus but somehow unable to train it properly. PFA the attached configuration of the model. Please suggest if any issue in the configuration.

[config.py.txt](https://github.com/rsennrich/nematus/files/1037429/config.py.txt)

The trained model is not even able to translate properly (translation output is "." only ).

**Comment by rsennrich** [comment]
Hello Raj,

the recommended way to use MRT is to first train a model with 
cross-entropy loss, and when this has converged, continue training with 
MRT. So far, our experience with MRT has been mixed, and we welcome more 
empirical data on good hyperparameters (so far, SGD with small learning 
rates seems the most robust) and effectiveness on different datasets.

best wishes,
Rico
